direct/nn/mri_models.py

```diff
@@ -20,7 +21,19 @@
 import direct.data.transforms as T
 from direct.config import BaseConfig
 from direct.engine import DoIterationOutput, Engine
-from direct.functionals import NMAELoss, NMSELoss, NRMSELoss, SobelGradL1Loss, SobelGradL2Loss, SSIMLoss
+from direct.functionals import (
+    HFENL1Loss,
+    HFENL2Loss,
+    NMAELoss,
+    NMSELoss,
+    NRMSELoss,
+    PSNRLoss,
+    SNRLoss,
+    SobelGradL1Loss,
+    SobelGradL2Loss,
+    SSIM3DLoss,
+    SSIMLoss,
+)
```

Comment:
Maybe it's better to do this `import direct.functionals as D`, and then do `D.SSIMLoss`, etc.